test/PrepaidCardMarketV2-test.js

```diff
@@ -788,6 +788,8 @@ contract("PrepaidCardMarketV2", (accounts) => {
         prepaidCardManager.address
       );
 
+      expect(createPrepaidCardEvent.issuer).to.be.equal(issuer);
```

Comment:
This is the regression test - without the change in contracts/PrepaidCardManager.sol, this assertion failed, which is clearly wrong.